play/src/front/ExternalModule/ExtensionModule.ts

```diff
@@ -61,3 +63,26 @@ export interface ExtensionModule {
     calendarSynchronised: boolean;
     todoListSynchronized: boolean;
 }
+
+export const RoomMetadataType = z.object({
+    player: z
+        .object({
+            accessTokens: z
+                .array(
+                    z.object({
+                        token: z.string(),
+                        provider: z.string(),
+                    })
+                )
+                .optional(),
+        })
+        .optional(),
+    modules: z.string().array(),
+    discordSettings: z.object({
```

Comment:
The 'Discord' module should not be identified in the core application. These properties should only be used on the external module.